src/pymmcore_gui/widgets/_toolbars.py

```diff
@@ -56,11 +56,16 @@ def __init__(
     ) -> None:
         super().__init__("Shutters", parent)
         self.mmc = mmc
+        # store all shutters widgets so that can be deleted when a new configuration is
+        # loaded
+        self._wdg: list[ShuttersWidget] = []
```

Comment:
can you achieve this without holding additional references to shutters widgets?  I would expect that Qt itself has some way to retrieve things added with `addWidget` (and I think it's probably just `self.actions`) ... did you find that `removeAction` wasn't sufficient?

Reply:
Yes, it is 👍. We just need to deal with the `UserWarnig` that will be emitted by each `ShutterWidget` itself. I'll fix it.